modules/widget-visibility/widget-conditions.php

```diff
@@ -443,24 +454,29 @@ public static function filter_widget( $instance ) {
 						}
 					break;
 					case 'tag':
-						if ( ! $rule['minor'] && is_tag() )
-							$condition_result = true;
-						else if ( is_singular() && $rule['minor'] && has_tag( $rule['minor'] ) )
+						if ( ! $rule['minor'] && is_tag() ) {
 							$condition_result = true;
-						else {
-							$tag = get_tag( $rule['minor'] );
-
-							if ( $tag && is_tag( $tag->slug ) )
+						} else {
+							$rule['minor'] = self::maybe_get_split_term( $rule['minor'], $rule['major'] );
+							if ( is_singular() && $rule['minor'] && has_tag( $rule['minor'] ) ) {
 								$condition_result = true;
+							} else {
+								$tag = get_tag( $rule['minor'] );
+								if ( $tag && is_tag( $tag->slug ) )
```

Comment:
can you add some <a href="https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style">braces to the conditionals</a>?